pyproject.toml

```diff
@@ -4,7 +4,7 @@ filename = "CHANGES.rst"
 directory = "CHANGES/"
 title_format = "{version} ({project_date})"
 template = "CHANGES/.TEMPLATE.rst"
-issue_format = "`#{issue} <https://pulp.plan.io/issues/{issue}>`_"
+issue_format = "`#{issue} <https://github.com/pulp/plugin_template/issues/{issue}>`_"
```

Comment:
I think, this change should come with an empty CHANGES directory.

Reply:
Yes and I need to coordinate with the open PRs too.